pkg/uploadproxy/uploadproxy.go

```diff
@@ -225,19 +225,20 @@ func (app *uploadProxyApp) testConnect(urlString string) error {
 		d := net.Dialer{Timeout: connectTimeout}
 		conn, err := d.Dial("tcp", hostPort)
 		if err == nil {
-			glog.Infof("Successfully connected to %s on attempt %d", urlString, i)
+			glog.V(3).Infof("Successfully connected to %s on attempt %d", urlString, i)
 			conn.Close()
 			return nil
 		}
 
 		switch ne := err.(type) {
 		case net.Error:
 			if ne.Timeout() {
+				glog.V(3).Infof("Timeout connecting to %s on iteration %d", hostPort, i)
 				continue
 			}
-			return errors.Wrapf(ne, "Unexpected net.Error dialing %s", urlString)
+			glog.V(3).Infof("Unexpected net error connecting to %s on iteration %d %+v", hostPort, i, err)
```

Comment:
Don't you want to use `glog` error message?

Reply:
These are transient/somewhat expected errors while the upload server pod starts.  Since upload proxy is a  long running process, don't want to spam the log.

Reply:
OK, I get that.